src/firebaseConfig.js

```diff
@@ -0,0 +1,12 @@
+const config = {
+    apiKey: "AIzaSyBMP9SytQq1810M77ln1TjN2gTT4T388iY",
+    authDomain: "react-to-do-8ff5f.firebaseapp.com",
```

Comment:
move these all keys into `env` and use them as a secret key, do not expose secret keys